src/pages/PatchRedirect.tsx

```diff
@@ -1,13 +1,60 @@
+import { useState, useEffect } from "react";
+import { useLazyQuery } from "@apollo/client";
 import { useParams, useLocation, Redirect } from "react-router-dom";
-import { getVersionRoute } from "constants/routes";
+import { PatchAndTaskFullPageLoad } from "components/Loading/PatchAndTaskFullPageLoad";
+import { commitQueueAlias } from "constants/patch";
+import {
+  getVersionRoute,
+  getPatchRoute,
+  getCommitQueueRoute,
+} from "constants/routes";
+import {
+  IsPatchConfigurableQuery,
+  IsPatchConfigurableQueryVariables,
+} from "gql/generated/types";
+import { GET_IS_PATCH_CONFIGURED } from "gql/queries";
 import { PatchTab } from "types/patch";
 import { queryString } from "utils";
+import { validatePatchId } from "utils/validators";
 
 const { parseQueryString } = queryString;
 
 export const PatchRedirect = () => {
   const { id, tab } = useParams<{ id: string; tab: PatchTab }>();
   const { search } = useLocation();
+  const [shouldRedirect, setShouldRedirect] = useState(false);
+  const [getPatch, { data }] = useLazyQuery<
+    IsPatchConfigurableQuery,
+    IsPatchConfigurableQueryVariables
+  >(GET_IS_PATCH_CONFIGURED, {
```

Comment:
"is patch configured" and "is patch configurable" seem to be saying two different things. the first says "is the patch finalized" and the second says "can i modify the tasks/variants for the patch". We should standardize this language.